.github/workflows/deploy.yml

```
@@ -0,0 +1,45 @@
+name: Deploy
+
+on: push
+
+env:
+  DOCKER_IMAGE: ${{ github.repository }}
+  DOCKER_REGISTRY: ghcr.io
+  DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+  DOCKER_USERNAME: ${{ github.repository_owner }}
+  HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Branch Tag
+        uses: nimblehq/branch-tag-action@v1.2
+
+      - name: Set HEROKU_APP
+        run: |
+          if [[ $BRANCH_TAG = "latest" ]]
+          then
+            echo "::set-env name=HEROKU_APP::go-google-scraper"
+          else
+            echo "::set-env name=HEROKU_APP::go-google-scraper-staging"
+          fi
+
+      - name: Docker login to docker container registry
+        run: "echo $DOCKER_TOKEN | docker login $DOCKER_REGISTRY -u $DOCKER_USERNAME --password-stdin"
+
+      - name: Docker build
+        run: |
+          docker-compose build
```

Tp use caching as [defined](https://github.com/gutakk/go-google-scraper/pull/19/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R17) in the Dockerfile, it's also necessary to pull already built images unless you plan to build new images each time 🤔 

```suggestion
		  docker-compose pull
          docker-compose build
```

---

I don't need to build new images each time. Fixed in 8d2b9647cb80cc63dd0b3d495febe0964a062007